### DISCUSSION TITLE
Saleor Marketing Analytics

### DISCUSSION THREAD

**Discussion by zdunecki** [saleor/saleor#15518] [Ideas]
Hi, 

As Saleor grows do you plan to support more robust marketing analytics solutions? I think a good reference to that is a [Shopify Web Pixel](https://shopify.dev/docs/apps/marketing/pixels) one. 

Thanks to that marketing providers can hook into your analytics solutions and move data to another destination via browser.

Best

**Comment by timuric** [comment]
Hi @zdunecki we don't have plans adding such solution, in our opinion analytics are separate projects with a lot of depth, we follow composable philosophy, to enable users to bring best of breed tools to their stack. You can find many open source tracking solutions, if you need something lightweight and simple check out https://plausible.io/

**Comment by zdunecki** [comment]
@timuric,
 
Thank you for your answer. Understandable.

Best
